datafusion/core/src/datasource/listing/mod.rs

```diff
@@ -103,3 +103,52 @@ impl From<ObjectMeta> for PartitionedFile {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
these tests are of `ListingTableUrl` which is in core/src/datasource/listing I moved them here